### ISSUE TITLE
Clicking on a new group in the Words menu scrolls the selected group off the of screen

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#1448]
To replicate:
Open an Eph/Words project
Scroll (if necessary) and click on the Christ, Messiah group
Scroll down in that group until the Create, Creation, Creator group appears near the bottom of the screen
Click on the Create, Creation, Creator group to open it
On my machine, the group at the top of my screen is now Forgive, Forgiveness
In order to see the group I clicked on, I need to scroll up quite a bit

**Comment by benjore** [comment]
Related to #1261 #1347 #1325 
Just ask Manny :)

**Comment by cckozie** [comment]
When a group is clicked, that group does remain in the visible screen area. However, the related issues noted above have not been addressed.

**Comment by RoyalSix** [comment]
Didn't see the referenced issues. On it.

**Comment by cckozie** [comment]
There are still problems. I followed the directions in the first comment:
1. Scrolled to and clicked on the Christ, Messiah group
2. Scrolled down in that group look for the Create, Creation, Creator group to appear. For some reason it is not currently in this project, so I clicked on the "cross" group when it was the third group from the bottom of the screen.
3. Now, the "gift" group is the top one on my screen. "cross" is a full page above it.
